storage/namespace.go

```diff
@@ -230,15 +265,33 @@ func (n *dbNamespace) Tick(softDeadline time.Duration) {
 	}
 
 	// Tick through the shards sequentially to avoid parallel data flushes
-	var r tickResult
+	var (
+		r  tickResult
+		l  sync.Mutex
+		wg sync.WaitGroup
+	)
 	perShardDeadline := softDeadline / time.Duration(len(shards))
+	perShardDeadline *= time.Duration(n.tickWorkersConcurrency)
```

Comment:
So the deadline is calculated for how much time each shard should get if we were ticking in a serial manner.  

The deadline is multiplied because we will be ticking concurrently, so essentially we are meeting the same `softDeadline` overall, but give each shard a little longer because they are being done concurrently.